modules/swagger-codegen-maven-plugin/src/main/java/io/swagger/codegen/plugin/CodeGenMojo.java

```diff
@@ -228,7 +228,11 @@
     @Parameter(name = "generateApiDocumentation", required = false)
     private Boolean generateApiDocumentation = true;
 
-
+    /**
+     * Skip the execution.
+     */
+    @Parameter(name = "skip", property = "codegen.skip", required = false, defaultValue = "false")
```

Comment:
I'm not sure about this name ... is `codegen.skip` a name used by other code generators too (where it might be sensible to skip all code generation in a project)?
Otherwise maybe `swagger-codegen.skip` would be more clear.